bundles/org.openhab.binding.dlinksmarthome/src/main/java/org/openhab/binding/dlinksmarthome/internal/handler/DLinkMotionSensorHandler.java

```diff
@@ -64,6 +64,9 @@ public void sensorStatus(final DeviceStatus status) {
             case ONLINE:
                 updateStatus(ThingStatus.ONLINE);
                 break;
+            case REBOOTING:
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.DUTY_CYCLE, "Device rebooting");
```

Comment:
ThingStatusDetail.DUTY_CYCLE is I believe not so much used but could be the right choice there.
Only used in 3 bindings: homematic, lutron and MQTT generic. I will see how it is used bu others.

Reply:
Are you happy with the use of DUTY_CYCLE?

Reply:
Let's assume it is OK.